src/components/Navigation/components/index.ts

```diff
@@ -49,6 +56,22 @@ export const StyledConnectWalletButton = styled.button`
     height: ${13 * GU}px;
     width: ${56 * GU}px;
   `}
+
+  ${(props) =>
+    props.connected &&
+    css`
```

Comment:
Do we need this `css` function?

Reply:
This just switches font size between when the button says "Connect" vs "0x12345...abcd". Though maybe it's cleaner to just make the address' `charsLength` 3. Fixed in 6925a12af2c57de963edc809022536cc114a9924

Reply:
Oh no, it was fine before! Just was wondering as we were already in a CSS context haha... And now with this new change I think it was better like it was before 😅

Reply:
Consider it reverted: 87cd257bb6a5f94667d45ff5ffc17d46a59f4772